web-client/src/primitives/hash.rs

```diff
@@ -30,4 +25,36 @@ impl Hash {
         nimiq_hash::Hasher::digest(nimiq_hash::sha512::Sha512Hasher::default(), data)
             .serialize_to_vec()
     }
+
+    #[wasm_bindgen(js_name = computeNimiqArgon2d)]
+    pub fn compute_nimiq_argon2d(
```

Comment:
```suggestion
    pub fn compute_nimiq_argon2d_kdf(
```

Reply:
Since we pass nimiq-specific parameters, and this is not the general-purpose argon2d, I think it's important to have `nimiq` in the name.